trollmoves/movers.py

```diff
@@ -201,13 +211,38 @@ class FtpMover(Mover):
     active_connections = dict()
     active_connection_lock = Lock()
 
+    def _get_netrc_authentication(self):
+        """Get login authentications from netrc file if available"""
+        LOGGER.debug("HOME = %s", os.environ.get('HOME'))
+        try:
+            secrets = netrc.netrc()
+        except (netrc.NetrcParseError, FileNotFoundError) as e__:
+            LOGGER.warning('Failed retrieve authentification details from netrc file! Exception: ', str(e__))
+            return
+
+        LOGGER.debug("hosts: %s", str(list(secrets.hosts.keys())))
```

Comment:
Is this log message necessary?

Reply:
Well, hopefully nut, I just needed extra debugging info during development